Libraries/WebSocket/WebSocketBase.js

```diff
@@ -33,19 +33,31 @@ class WebSocketBase extends EventTarget {
   readyState: number;
   url: ?string;
 
-  constructor(url: string, protocols: ?any) {
+  /**
+   * @constructor WebSocket
+   * @param {String} address Connection address.
+   * @param {String|Array} protocols WebSocket protocols.
+   * @param {Object} options Additional connection options.
```

Comment:
Might want to define a typedef for the options so it's clear exactly what options are supported.